src/black/linegen.py

```diff
@@ -812,35 +813,40 @@ def maybe_make_parens_invisible_in_atom(node: LN, parent: LN) -> bool:
     ):
         return False
 
-    if is_walrus_assignment(node):
-        if parent.type in [
-            syms.annassign,
-            syms.expr_stmt,
-            syms.assert_stmt,
-            syms.return_stmt,
-            # these ones aren't useful to end users, but they do please fuzzers
-            syms.for_stmt,
-            syms.del_stmt,
-        ]:
-            return False
+    if is_walrus_assignment(node) and parent.type in [
+        syms.annassign,
+        syms.expr_stmt,
+        syms.assert_stmt,
+        syms.return_stmt,
+        # these ones aren't useful to end users, but they do please fuzzers
+        syms.for_stmt,
+        syms.del_stmt,
+    ]:
+        return False
 
     first = node.children[0]
     last = node.children[-1]
     if first.type == token.LPAR and last.type == token.RPAR:
-        middle = node.children[1]
-        # make parentheses invisible
-        first.value = ""  # type: ignore
-        last.value = ""  # type: ignore
-        maybe_make_parens_invisible_in_atom(middle, parent=parent)
+        return _extracted_from_maybe_make_parens_invisible_in_atom_28(
+            node, first, last, parent
+        )
 
-        if is_atom_with_invisible_parens(middle):
-            # Strip the invisible parens from `middle` by replacing
-            # it with the child in-between the invisible parens
-            middle.replace(middle.children[1])
+    return True
 
-        return False
 
-    return True
+def _extracted_from_maybe_make_parens_invisible_in_atom_28(node, first, last, parent):
```

Comment:
What does 28 mean here?

Reply:
Sorry, the number `28` is just a filler for the help-functions, but actually not needed